examples/dcgan_fashiongen/create_mar.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+REPONAME="pytorch_GAN_zoo"
+SHA="b75dee40918caabb4fe7ec561522717bf096a8cb" #master branch as of 27th Aug 2020
+SRCZIP="$SHA.zip"
+SRCDIR="$REPONAME-$SHA"
+MODELSZIP="models.zip"
+MODELSDIR="models"
+CHECKPOINT="DCGAN_fashionGen-1d67302.pth" #The DCGAN pretrained model as of 27th Aug 2020
+CHECKPOINT_RENAMED="DCGAN_fashionGen.pth"
+
+# Clean Up before exit
+function cleanup {
+  rm -rf $SRCZIP $SRCDIR $MODELSZIP $CHECKPOINT_RENAMED $MODELSDIR
+}
+trap cleanup EXIT
+
+# Download and Extract model's source code
+sudo apt-get install zip unzip
```

Comment:
* We should specify in the docs that this example works only on *nix with apt as the package manager
* Or make it platform agnostic by having the customer install the dependency with appropriate instructions in the README